custom_components/reflex_clerk_api/clerk_provider.py

```
@@ -84,6 +86,29 @@ def set_claims_options(cls, claims_options: dict[str, Any]) -> None:
+        due to clock drift between Clerk servers and your backend.
+
+        Args:
+            seconds: Non-negative integer, max 3600 (1 hour).
+
+        Raises:
+            ValueError: If seconds is negative or exceeds 3600.
+        """
+        if not isinstance(seconds, int) or seconds < 0:
+            raise ValueError(
+                f"jwt_validate_leeway_seconds must be a non-negative integer, got {seconds!r}"
+            )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)